bip-ephemeralanchors.mediawiki

```diff
@@ -0,0 +1,123 @@
+<pre>
+  BIP: ?
+  Layer: Application
+  Title: Ephemeral Anchors
+  Author: Gregory Sanders <gsanders87@gmail.com>
+  Status: Draft
+  License: BSD-3-Clause
```

Comment:
The proposal is missing the mandatory copyright section. Since BSD-3 is a software license, perhaps consider a CC0 1.0 Universal license.

Reply:
sure